libindy/tests/utils/mod.rs

```diff
@@ -1,7 +1,11 @@
-#![allow(dead_code)]
+#![allow(dead_code, unused_macros)]
```

Comment:
@mikelodder7 it's better to use `allow(some_warning)` for minimal context, like near module import and without `!` (if possible). But we can do it later and do not block the PR